documentation/SDKs/javascript/README.md

```diff
@@ -7,352 +7,264 @@
 ### Installing the JavaScript SDK for Nether
 
 ## Usage
-
-### Analytics
-
-#### Active sessions - daily
-You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsDaily();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Day + '-' + sessions[0].Month + '-' + sessions[0].Year);
-```
-#### Active sessions - monthly
-You can use the ***Nether.Analytics.GetActiveSessionsMonthly*** method to get the total active sessions per month. Returns an array of ActiveSession
-##### Example usage
-```javascript
-var sessions = Nether.Analytics.GetActiveSessionsMonthly();
-console.log(sessions[0].ActiveSessions + ' for ' + sessions[0].Month + '-' + sessions[0].Year);
+To use the nether SDK include the nether script in your project. Reference the script in the webpage using 
+```html
+<script src="/scripts/nether.js"></script>
+```
+```javascript
+var config = {
+        netherClientId: '<client Id>',
+        netherClientSecret: '<client secret>',
+        facebookAppId: '<facebook app Id>',
+        netherBaseUrl: '<nether url>'
+    }
+    nether.init(config, facebookInitailised, netherInitialised);
```

Comment:
There are some things that need to be updated, but I like the direction where we are heading and just the fact that we can now start seeing how I (as a JavaScript game developer) can interact with Nether will probably evolve into more changes.

I agree with Stuart that we should try and keep authentication as open as possible and give the option of either login with a NetherToken or with FacebookToken. I think that is the most important work we need to do with the SDK at this stage.

I would also love the ability to send events to the Analytics pipeline without being logged on, but I think we need to wait with that functionality until we have implemented "guest user", since we don't want to open up EventHub for any unauthenticated user to post messages to. So let's keep it as this for now and we'll think about that workflow.

There are a bunch of things going on with the analytics pipeline of Nether so some of the messages in there might not be valid anymore, but I think we should keep the implementation as is and more use it as a template on how to implement the "actual supported messages" going forward.

I can see that we do expose some APIs that doesn't make any sense for the game client to call. For example, I don't think we should have the ability to drop all my posted scores from the client. Probably we should separate the APIs on the server side so it makes more sense, what APIs are intended for the game client and what are intended for server-to-server communication (i.e. admin APIs).